test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -660,6 +670,9 @@ ucs_status_t UcxContext::am_recv_callback(void *arg, const void *header,
 
     self->dispatch_am_message(conn, header, header_length, data_desc);
 
+    /* If data is going to be used outside this callback, UCS_INPROGRESS
+     * should be returned. Call ucp_am_data_release() when data is not needed.
+     */
```

Comment:
add to the comment that in the current example, dispatch_am_message is always processing the received data internally and never needs it later

Reply:
OK

Reply:
done